src/Compilers/CSharp/csc/csc.csproj

```diff
@@ -57,7 +54,7 @@
   </ItemGroup>
   <ItemGroup>
     <None Include="App.config" />
-    <None Include="csc.rsp">
+    <None Include="csc.rsp" Condition="'$(TargetFramework)' != 'netcoreapp2.0'">
```

Comment:
This is a bit scary... seems like coreclr's csc might have been broken since https://github.com/dotnet/roslyn/pull/22142 merged. However, I think we don't package csc.rsp in the netcore nuget packages, so the nuget packages should be fine... still, fixing this asap is good :)

Reply:
I would invert this condition -- only copy if net46 is true. That way if we change `netcoreapp2.0`  to something else later the essential logic is the same: `csc.rsp` is only meaningful on desktop.